front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleConfig.tsx

```diff
@@ -203,13 +199,40 @@ const ImportTrainScheduleConfig = ({
         const formattedArrivalTime = `${startDate} ${arrivalTime}`;
         const formattedDepartureTime = `${startDate} ${departureTime}`;
 
+        let stopFor: number | undefined;
+
+        const arrivalDate = new Date(`${startDate}T${arrivalTime}`);
+        const departureDate = new Date(`${startDate}T${departureTime}`);
+        if (ocpType === 'stop') {
+          if (arrivalTime && departureTime) {
+            if (arrivalTime === departureTime) {
+              stopFor = 0;
+            } else {
+              stopFor = Math.round((departureDate.getTime() - arrivalDate.getTime()) / 1000);
+            }
+          } else if (departureTime) {
+            stopFor = 0;
+          }
```

Comment:
Do you think you can handle the case this way:
- if the stop has a departure and a arrival times, then compute the stop duration
- else set the stop duration to 0
```suggestion
          } else {
            stopFor = 0;
          }
```